### DISCUSSION TITLE
specify different image file for catalog grid than for object entry?

### DISCUSSION THREAD

**Discussion by bjhewitt** [thegetty/quire#898] [Get Help]
I have an object catalog image grid. All the object entry pages use HUGE images as their key image because those are set to use the IIIF zoom. This, unfortunately, means the catalog grid will be very slow loading with dozens of those very large images pulled in. Is there a way to specify a different image to use in the grid than the object entry key image? (Or another workaround?)

**Comment by Erin-Cecele** [comment]
Hi @bjhewitt, this is a great question. We are releasing a new "object grid" feature in the next few weeks that will solve this issue.  This feature provides a higher level of control over the way you display objects in a TOC grid. This includes using images that are downsized (provided they started as IIIF images which would be the case for you.) I hope to have documentation ready and announced in the next newsletter going out on 2/15. Please stay tuned!
